grails-app/services/com/unifina/service/StreamService.groovy

```diff
@@ -27,6 +27,7 @@ class StreamService {
 
 	CassandraService cassandraService
 	PermissionService permissionService
+	EthereumIntegrationKeyService ethService
```

Comment:
There is a convention to use full name for the variable name. There also is an EthereumService class.
Rename `ethService` -> `ethereumIntegrationKeyService`